code/obj/machinery/computer/announcement.dm

```diff
@@ -225,9 +225,12 @@
 		var/job = person.mind.assigned_role
 		if(!job || job == "MODE")
 			job = "Staff Assistant"
+		if(issilicon(person))
+			job = "Cyborg"
```

Comment:
Do cyborgs not have an assigned role?

Reply:
Not sure to be fair, but this was my first try at repairing a bug and i mostly followed instructions of people from the discord server. In the end i was resulted with this, tested it and it worked. 😄